examples/with-jest-react-testing-library/package.json

```
@@ -0,0 +1,22 @@
+{
+  "name": "with-jest-react-testing-library",
+  "version": "1.0.0",
+  "dependencies": {
+    "next": "^7.0.2",
```

Let's use `latest` like the other examples:

```suggestion
     "next": "latest",
```